webpack/ForemanTasks/Components/TasksTable/__tests__/__snapshots__/TasksTablePage.test.js.snap

```diff
@@ -227,7 +227,7 @@ exports[`TasksTablePage rendering render with minimal props 1`] = `
         />
         <ExportButton
           title="Export All"
-          url="/foreman_tasks/tasks.csv?action=%22some-name%22&state=stopped"
+          url="/foreman_tasks/tasks.csv?action=%22some-name%22&search=%28state%3Dstopped%29"
```

Comment:
This doesn't look like the correct query, shouldn't it be `search=(state=stopped& action=some-name)?

Reply:
Its happening because getURIQuery is mocked and ` history.location.search` is no longer used in `getCSVurl` so its ok